packages/core/src/connectors/types.ts

```diff
@@ -3,9 +3,9 @@ import { ConnectorType } from '@logto/schemas';
 export interface ConnectorMetadata {
   id: string;
   type: ConnectorType;
-  name: string;
+  name: Record<string, string>;
```

Comment:
Record <enum, string> ?

Reply:
We currently don't have a language code enums in this project, keep the same for now?

Reply:
k

Reply:
actually `keyof resource` do the trick. would you like to add an export like `type Languages = keyof resource;` in `packages/phrases/src/index.ts`?